dockstore-client/bin/dockstore

```diff
@@ -235,13 +235,14 @@ else
       exit -100;
     fi
 
-    if [[ "$_java" ]]; then
-      version=$("$_java" -version 2>&1 | awk -F '"' '/version/ {print $2}')
-      if [[ "$version" < "1.8" ]]; then
-        echo "Dockstore requires Java version 1.8 and above."
-        exit -100;
-      fi
-    fi
+# does not work with OpenJDK 10 on Ubuntu. format looks very different
```

Comment:
Did you mean to leave this commented out? There is no longer a check for >= 1.8, is there?

Reply:
yeah, I think the check will need to be fixed or improved, but isn't close to working in its current incarnation